modules/nodes-sigstore-configure-image-policy.adoc

```diff
@@ -212,15 +215,16 @@ sh-5.1# cat /etc/crio/policies/<namespace>.json
 sh-5.1# cat /etc/containers/registries.d/sigstore-registries.yaml
 ----
 +
-.Example output
+.Example output showing that the scoped registry was added
 [source,yaml]
 ----
 docker:
   example.com:
```

Comment:
In this example, the repo should be: example.io/crio/signed

Reply:
this is still outstanding